packages/components/src/dashicon/index.js

```diff
@@ -913,6 +913,7 @@ export default class Dashicon extends Component {
 				width={ size }
 				height={ size }
 				viewBox="0 0 20 20"
+				{ ...extraProps }
```

Comment:
It's also important to raise that this component won't get re-rendered where any of the extra props passed changes because of this component implements `shouldComponentUpdate`. This function narrows down props changed check to `icon`, `size`, `className` and `ariaPressed`.

Reply:
Thanks for the heads-up!
We are discussing this same point here https://github.com/WordPress/gutenberg/pull/13977#discussion_r261677967 👍